pennylane/transforms/hamiltonian_expand.py

```diff
@@ -189,3 +192,42 @@ def processing_fn(res):
         return qml.math.sum(qml.math.stack(dot_products), axis=0)
 
     return tapes, processing_fn
+
+
+def sum_expand(tape: QuantumScript):
+    """Splits a tape measuring a Sum expectation into mutliple tapes of summand expectations,
+    and provides a function to recombine the results.
+
+    Args:
+        tape (.QuantumTape): the tape used when calculating the expectation value
+            of the Hamiltonian
+
+    Returns:
+        tuple[list[.QuantumTape], function]: Returns a tuple containing a list of
+        quantum tapes to be evaluated, and a function to be applied to these
+        tape executions to compute the expectation value.
+    """
+    sum_op = tape.measurements[0].obs
+    if (
+        not isinstance(sum_op, Sum)
```

Comment:
What if there is a summed op _and_ a single op? Or is this only used in cases where it is assumed to already be split into individual expectation values?

Reply:
As mentioned below, right now this only accepts one measurement. But we can definitely expand it to multiple measurements if needed.

Reply:
If its not a big problem it would definitely be great to be able to output multiple Sums (or combinations with other observables)!

Reply:
Done!